CONTRIBUTING.md

```diff
@@ -6,7 +6,7 @@ just a few small guidelines you need to follow.
 ## Contributor License Agreement
 
 Contributions to this project must be accompanied by a Contributor License
-Agreement. You (or your employer) retain the copyright to your contribution;
+Agreement. You (or your employer) retain the copyright to your contribution; 
```

Comment:
```suggestion
Agreement. You (or your employer) retain the copyright to your contribution;
```
trailing space (editor can trim on save)

Reply:
Configured! Thanks for teaching me to fish :-D